Plugwise_Smile/Smile.py

```diff
@@ -136,10 +136,11 @@ async def _create_session() -> aiohttp.ClientSession:
         self.dsmrmain_id = None
         self.gateway_id = None
         self.heater_id = None
-        self.hostname =  None
+        self.hostname = None
```

Comment:
Maybe name this `self.smile_hostname`? To be in line with the other `smile_`variables?

Reply:
Adjusting